internal/connector/exportable_resource.go

```diff
@@ -23,3 +25,17 @@ type ExportableResource interface {
 	ExportAll() (*[]ImportBlock, error)
 	ResourceType() string
 }
+
+func (b *ImportBlock) Sanitize() {
+
+	// Replace spaces with underscores
+	b.ResourceName = strings.ReplaceAll(b.ResourceName, " ", "_")
+	b.ResourceType = strings.ReplaceAll(b.ResourceType, " ", "_")
```

Comment:
It seems like ResourceType should be predictable, and we shouldn't have to sanitize it, right? Since it's always a hard-coded string in the resource implementation?

Reply:
removed!